### PR TITLE
implemented replay button in ScenarioOverActivity

### PR DESCRIPTION
**Pull request by jacqueline8711**
–> #167

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/ScenarioOverActivity.java [hunk 1]
```diff
@@ -114,6 +114,20 @@ public void onClick(View v) {
...
+            @Override
+            public void onClick(View view) {
+                SessionHistory.currSessionID = SessionHistory.prevSessionID;
+                SessionHistory.totalPoints -= SessionHistory.currScenePoints;
+                SessionHistory.currScenePoints = 0;
+                scenarioActivityDone = 0;
+                DatabaseHandler dbHandler = new DatabaseHandler(ScenarioOverActivity.this);
+                dbHandler.resetCompleted(SessionHistory.currSessionID);
+                dbHandler.resetReplayed(SessionHistory.currSessionID);
+                startActivity(new Intent(ScenarioOverActivity.this, GameActivity.class));
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/ScenarioOverActivity.java]: So a new activity is launched here without finishing the last one which leads to the same scene being created as many times as replay is clicked. Multiple games should not be present in the back-stack. Please fix.

**Comment by jacqueline8711** [comment]
I changed it and I think I got it right now, but I'm not sure.

**Comment by chhavip** [comment]
Nice work! Approving the work done.

**Comment by chhavip** [comment]
Closing to choose after #GCI

**Comment by chhavip** [comment]
Have re-opened this PR, please check for correctness as it looks good to me @anubhakushwaha @aanchal07

**Comment by chhavip** [comment]
Please change the target branch to develop instead of master for merge @jacqueline8711

**Comment by jacqueline8711** [comment]
@chhavip done

**Comment by chhavip** [comment]
@jacqueline8711 The PR needs to be rebased as it now has changes from the recent changes in the develop branch. Please make sure there is only one commit for the change.

**Comment by jacqueline8711** [comment]
@chhavip I rebased it

**Comment by chhavip** [comment]
Merging.

**Pull request merged**